zone/corpse.h

```diff
@@ -160,6 +168,7 @@ class Corpse : public Mob {
 	Timer		corpse_graveyard_timer;
 	Timer		loot_cooldown_timer; /* Delay between loot actions on the corpse entity */
 	EQEmu::TintProfile item_tint;
+	std::vector<std::string> consent_names;
```

Comment:
Would it make more sense for this to be `consented_player_names` ? I could mistake this as a few different things